tools/eslint-plugin/dist/configs/no-internal.js

```diff
@@ -0,0 +1,11 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+module.exports = {
+  "plugins": [ "@bentley" ],
+  "extends": "plugin:@bentley/imodeljs-recommended",
+  "rules": {
+    "@bentley/no-internal": [ "error", { "tag": [ "internal", "alpha", "beta" ]}]
```

Comment:
The default in the rule is `[ "internal", "alpha"]`. I think that and the config here should be made consistent by removing the "tag" here.
I'm not sure which one makes more sense for most people, but if you think including beta is useful, we can probably change that. @calebmshafer, any thoughts?

Reply:
I get that the output is grouped anyway so it's fairly easy to just ignore the tags you don't care about, but if you're putting this in the `configs` dir, you're exposing it alongside the other configs in the plugin. So maybe a better option would be to use the `--rule` option instead of a config file, or put the config file in a different directory

Reply:
Great suggestion, I've removed the file from configs and am using the -c, --plugin and --rule options instead.